### ISSUE TITLE
AMP Training

### ISSUE THREAD

**Issue by egeozsoy** [lucidrains/DALLE2-pytorch#82]
There is a AMP flag, but has this feature been tested at all? For me it does not work

**Comment by lucidrains** [comment]
@egeozsoy i see the issue! (and put in a fix) interesting the amp autocast didn't take care of this :thinking:

**Comment by egeozsoy** [comment]
Yes was also surprised, I think it might be related to the fact that you are not doing a forward_call, but just using the parameter as it. Still strange though...
